2-extra/1-quote-generator-revisited/quotes.js

```diff
@@ -0,0 +1,516 @@
+    quote:
+      "Remember that not getting what you want is sometimes a wonderful stroke of luck.",
+    author: "Dalai Lama",
+  },
+  {
+    quote: "You can’t use up creativity.  The more you use, the more you have.",
+    author: "Maya Angelou",
+  },
+  {
+    quote: "Dream big and dare to fail.",
+    author: "Norman Vaughan",
+  },
+  {
+    quote:
+      "Our lives begin to end the day we become silent about things that matter.",
+    author: "Martin Luther King Jr.",
+  },
+  {
+    quote: "Do what you can, where you are, with what you have.",
+    author: "Teddy Roosevelt",
+  },
+  {
+    quote:
+      "If you do what you’ve always done, you’ll get what you’ve always gotten.",
+    author: "Tony Robbins",
+  },
+  {
+    quote: "Dreaming, after all, is a form of planning.",
+    author: "Gloria Steinem",
+  },
+  {
+    quote:
+      "It’s your place in the world; it’s your life. Go on and do all you can with it, and make it the life you want to live.",
+    author: "Mae Jemison",
+  },
+  {
+    quote:
+      "You may be disappointed if you fail, but you are doomed if you don’t try.",
+    author: "Beverly Sills",
+  },
+  {
+    quote: "Remember no one can make you feel inferior without your consent.",
+    author: "Eleanor Roosevelt",
+  },
+  {
+    quote: "Life is what we make it, always has been, always will be.",
+    author: "Grandma Moses",
+  },
+  {
+    quote:
+      "The question isn’t who is going to let me; it’s who is going to stop me.",
+    author: "Ayn Rand",
+  },
+  {
+    quote:
+      "When everything seems to be going against you, remember that the airplane takes off against the wind, not with it.",
+    author: "Henry Ford",
+  },
+  {
+    quote:
+      "It’s not the years in your life that count. It’s the life in your years.",
+    author: "Abraham Lincoln",
+  },
+  {
+    quote: "Change your thoughts and you change your world.",
+    author: "Norman Vincent Peale",
+  },
+  {
+    quote:
+      "Either write something worth reading or do something worth writing.",
+    author: "Benjamin Franklin",
+  },
+  {
+    quote: "Nothing is impossible, the word itself says, “I’m possible!”",
+    author: "–Audrey Hepburn",
+  },
+  {
+    quote: "The only way to do great work is to love what you do.",
+    author: "Steve Jobs",
+  },
+  {
+    quote: "If you can dream it, you can achieve it.",
+    author: "Zig Ziglar",
+  },
+];
+const quoteP = document.querySelector("#quote");
+const authorP = document.querySelector("#author");
+const quoteBtn = document.querySelector("#new-quote");
+const autoPlayBtn = document.querySelector("#auto-play");
+let autoPlayInterval;
+function random() {
+  let num = Math.floor(Math.random() * quotes.length);
+  const { quote, author } = quotes[num];
+  quoteP.textContent = quote;
+  authorP.textContent = author;
+}
+function autoPlay(e) {
+  if (e.target.textContent === "auto-play:OFF") {
+    e.target.textContent = "auto-play:ON";
+    autoPlayInterval = setInterval(random, 1000);
```

Comment:
The interval should be 60 seconds, but your logic is correct, so that's fine

Reply:
Oh...I forgot to correct when I checked, because no one will wait 60 sec while debugging...